enhancements/ingress/set-delete-http-headers.md

```diff
@@ -742,8 +745,15 @@ So, the custom HTTP request value of `x-forwarded-for` header set using `spec.ht
 or `spec.httpHeaders.actions[*].request` of Ingress Controller as in case of HTTP request headers, HAProxy overrides the value of the frontend of same header name set by IngressController with the
 value which was present in the backend section.
 
-Another use case is deleting the header which forwardedHeaderPolicy does not provide but this EP does which is the reason why we will allow users to set/delete
-`x-forwarded-for` headers using this API.
+This enhancement provides a method for a user to  delete the `x-forwarded-for` header, which the Ingress Controller `spec.httpHeaders.forwardedHeaderPolicy` does not provide.
+
+
+| values overriden by spec.httpHeaders.action &rarr;<br/>values set along with spec.httpHeaders.action &darr;                                                                      | haproxy.router.openshift.io/set-forwarded-headers | spec.httpHeaders.forwardedHeaderPolicy |
```

Comment:
I don't know why you added this chart, I don't find any place where someone asked for it.  I also don't know what this chart means. It says "yes" in four places, but I don't know what question that answers.  😵‍💫   If you want to keep it, please add an introduction to what it means and clarify what each table column means.  I think it's better to delete it to move this along.

Reply:
Hi @candita ! You had asked for it. I will try to find out that comment. I am OK to remove this. I am not sure what is confusing you. I will add an introduction.

Reply:
https://github.com/openshift/enhancements/pull/1446#discussion_r1313260984

Reply:
This is the kind of chart I was talking about.  Can you use this and fill out the third column?  I still don't know the answer.

| When this combination of values is present                                                                             | This is the value that is used                      |
|------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------|
| haproxy.router.openshift.io/set-forwarded-headers, spec.httpHeaders.forwardedHeaderPolicy, and spec.httpHeaders.action | spec.httpHeaders.action?                            |
| haproxy.router.openshift.io/set-forwarded-headers and spec.httpHeaders.forwardedHeaderPolicy                           | haproxy.router.openshift.io/set-forwarded-headers ? |
| haproxy.router.openshift.io/set-forwarded-headers and spec.httpHeaders.action                                          | ?                                                   |
| spec.httpHeaders.forwardedHeaderPolicy and spec.httpHeaders.action                                                     | spec.httpHeaders.action                             |